app/javascript/react/screens/App/Overview/helper.js

```diff
@@ -0,0 +1,16 @@
+import getMostRecentRequest from '../common/getMostRecentRequest';
+
+export const planTransmutation = (plans = [], mappings = []) =>
+  plans.map(plan => {
+    const infraMappingName = mappings.find(
+      mapping => mapping.id === plan.options.config_info.transformation_mapping_id
+    );
+    const status = getMostRecentRequest(plan.miq_requests);
```

Comment:
another minor nit -

Can you rename this variable to `request` or something like that? (since it's really a request)